### DISCUSSION TITLE
Is search available

### DISCUSSION THREAD

**Discussion by ValMystletainn** [jupyter-book/mystmd#1554] [Q&A] [answered]
I have updated the myst cli to ver 1.3.11, and find that on the right of the upper bar is a search box. And find some pr about the search. But I have not found the any documents about it. Is this feature ready to be used?

**Comment by agoose77** [comment, marked as answer]
Hi @ValMystletainn - yes, search is ready to try out! We expect some bugs at this early stage, so do let us know if you notice anything.

We will be adding some documentation on the standards behind our search, but probably no documentation (it should be fairly simple!)
